src/helpers/getPetitionLink.js

```diff
@@ -1,7 +1,11 @@
-export default (id) => {
-  return [
-    process.env.BASE_URL,
-    'petitions',
-    id
-  ].join('/');
+export default (id, fullURL) => {
+  let link = ['petitions', id];
+
+  if (fullURL) {
+    link.unshift(process.env.BASE_URL);
+  } else {
+    link[0] = `/${link[0]}`;
+  }
```

Comment:
Oh, Github doesn’t hide the comments on old diffs with the new reviews, but this is done. Yes, good solution. :)